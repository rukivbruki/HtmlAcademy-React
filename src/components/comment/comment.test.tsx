import * as React from "react";
import * as renderer from "react-test-renderer";
import Comment from './comment';

const props = {
  user: {
    id: 1,
    name: ``,
  },
  rating: 1,
  comment: ``,
  date: `11.11.11`
};

it(`Comment renders correctly`, () => {
  const tree = renderer
    .create(
        <Comment {...props}/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
