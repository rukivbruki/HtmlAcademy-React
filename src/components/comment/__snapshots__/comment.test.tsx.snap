// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment renders correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
      style={
        Object {
          "wordBreak": "break-word",
        }
      }
    >
      
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        
      </cite>
      <time
        className="review__date"
        dateTime="11.10.2011"
      >
        November 11, 2011
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    1
  </div>
</div>
`;
